ember-server/shared/src/test/scala/org/http4s/ember/server/EmberUnixSocketSuite.scala

```diff
@@ -46,8 +46,8 @@ class EmberUnixSocketSuite extends Http4sSuite {
         .build
         .map(UnixSocket(localSocket))
       _ <- Resource.eval(IO.sleep(4.seconds))
-      resp <- client.run(Request[IO](Method.GET))
-      body <- Resource.eval(resp.bodyText.compile.string)
+      resp <- client.run(Request(Method.GET))
+      body <- Resource.eval(resp.bodyText[IO].compile.string)
```

Comment:
This is annoying. Scala 3 wants it, but not 2.13.

Reply:
Yeah, but I'd much rather have it needed where something effectful is actually being done.